requirements.txt

```diff
@@ -1 +1,4 @@
-
+beautifulsoup4==4.10.0
+requests==2.27.1
```

Comment:
order

Reply:
fix that by next Wednesday, or you will have a penalty of -1 for your mark for scrapper. You should have done that during the hand over process!

Reply:
Fixed import and forgot about this one. Now done.